src/news/2016-03-28-upcoming-talks-on-inclusive-digital-education.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Upcoming Talks on Inclusive Digital Education
+date: '2016-03-28'
+---
+IDRC Director Jutta Treviranus will be speaking at the following conferences
+on topics crossing digital inclusion in education, to rethinking the role of technology
+in our institutions.
+April 2016:
+
+- April 12 -[WWW2016 Workshop onWeb Science and Technology for Education (WWW2016 Montreal)](https://events.keep.edu.hk/webed/2016/)
```

Comment:
The formatting of this list is incorrect, please make it consistent with the original page